src/Components/Projects/Projects.tsx

```diff
@@ -76,13 +210,11 @@ export default function Projects() {
               <tr key={project?.id}>
                 <td scope="row"> {index + 1} </td>
                 <td> {project?.title} </td>
-                <td className=""> <div className="status w-50 text-center rounded-5"> <span>Public</span> </div></td>
-                <td> 10 </td>
-                <td> 10 </td>
-                <td> 20/12/2023 </td>            
+                <td> {project?.description} </td>       
+                <td> {project?.creationDate.slice( 0 , 10)} </td>            
                 <td className='text-center'>
-                  <i className='fa fs-6 text-success fa-edit'></i>
-                  <i className='fa ms-3 fs-6 text-danger fa-trash'></i>
+                  <i onClick={()=> showUpdateModel (project)} className='fa fs-6 text-success fa-edit'></i>
```

Comment:
There is a space between the function and the parentheses

Reply:
We can make an icon clickable by adding onClick event to it, but this is not a good practice. and we are losing a lot of features by not using a button for that like using tap to navigate between interactive elements, and pressing space to click it, and more.

You should instead make a button and nest the icon inside it, and the button should be giving the onClick event.

below is an illustration of some of the accessibility issues that are caused by not using a button:

before (the view icon is not accessible via the tap key nor it can be clicked using space):
![before using a button](https://github.com/AhmedEl-Hussin/PMS/assets/64327685/cad05ef9-9d12-4801-be03-745dfc6fa56c)

after (the view icon is accessible via the tap key nor it can be clicked using space):
![after using a button](https://github.com/AhmedEl-Hussin/PMS/assets/64327685/17c3a954-f4ce-4322-bace-f869da887f74)